fairmq/shmem/Manager.h

```diff
@@ -430,7 +430,7 @@ class Manager
     std::unordered_map<uint64_t, std::unique_ptr<Region>> fRegions;
 
     std::atomic<bool> fInterrupted;
-    std::atomic<int32_t> fMsgCounter; // TODO: find a better lifetime solution instead of the counter
+    std::atomic<int64_t> fMsgCounter; // TODO: find a better lifetime solution instead of the counter
```

Comment:
Why 64-bit? This increases the `sizeof` of this class by 8 bytes (for lock-free atomics) and is potentially a portability issue if anyone ever wants to build for a 32-bit system.

Reply:
32bit should be enough, even for our application. Otherwise we're really doing something wrong... There should be 1 instance of this per process, so +8 B is ok even in this economy :)